mgit/README.md

```diff
@@ -0,0 +1,87 @@
+# MGit
+
+Tool for management the multi-repository architecture.
+
+## Instalation
+
+```bash
+npm install @cksource/mgit
+```
+
+## Usage
+
+Firstly, you need to create a configuration file `mgit.json`:
+
+```json
+{
+  "packages": "packages/",
+  "dependencies": {
+    "package/path": "github/url"
+  }
+}
+```
+
+* `packages` - the directory where dependencies will be cloned,
+* `dependencies` - list of dependencies.
+
+The following dependencies are proper and can be placed into configuration file:
+
+Package will be installed in `packages/@scope/package` from branch `develop`:
+
+```json
+{
+	"@scope/package": "scopes/package#develop"
+}
+```
+
+Package will be installed in `packages/package` from branch `master`:
+
+```json
+{
+	"package": "https://github.com/github/package"
```

Comment:
This is not a correct form – you shouldn't clone by `https://` if you want to work on this package (you won't be able to push). Only the GH URL (`org/repo`) or full git URL are correct forms.